src/components/Textarea/Textarea.tsx

```diff
@@ -29,92 +24,11 @@ export type TextareaProps = {
-            {decorator && (
-                <div
-                    className="tw-absolute tw-top-2 tw-left-2 tw-inline-flex tw-items-end tw-text-black-80"
-                    data-test-id="decorator"
-                >
-                    {decorator}
-                </div>
-            )}
-            <Component
-                {...(mergeProps(focusProps, {
-                    onBlur: (event: FocusEvent<HTMLTextAreaElement>) => onBlur && onBlur(event.target.value),
-                    onInput: (event: FormEvent<HTMLTextAreaElement>) =>
-                        onInput && onInput((event.target as HTMLTextAreaElement).value),
-                }) as TextareaAutosizeProps)}
-                {...(autosize ? autosizeProps : { rows: minRows })}
-                id={useMemoizedId(propId)}
-                value={value}
-                placeholder={placeholder}
-                required={required}
-                className={merge([
-                    'tw-w-full tw-p-2 tw-border tw-rounded tw-text-s tw-outline-none tw-transition tw-placeholder-black-60',
-                    !!decorator && 'tw-pl-7 ',
-                    disabled
-                        ? 'tw-border-black-5 tw-bg-black-5 tw-text-black-40'
-                        : 'tw-text-black tw-border-black-20 hover:tw-border-black-90',
-                    isFocusVisible && FOCUS_STYLE,
-                    validationClassMap[validation],
-                    !resizeable && 'tw-resize-none',
-                    validation === Validation.Error && 'tw-pr-8',
-                ])}
-                disabled={disabled}
-                onFocus={(e) => {
-                    if (selectable) {
-                        e.target.select();
-                    }
-                    if (onFocus) {
-                        onFocus(e);
-                    }
-                }}
-                data-test-id="textarea"
-            />
-            {validation === Validation.Loading && (
-                <span className="tw-absolute tw-top-[-0.55rem] tw-right-[-0.55rem] tw-bg-white tw-rounded-full tw-p-[2px] tw-border tw-border-black-10">
-                    <LoadingCircle size={LoadingCircleSize.ExtraSmall} />
-                </span>
-            )}
-            {(validation === Validation.Error || validation === Validation.Warning) && (
-                <span
-                    className={merge([
-                        'tw-absolute tw-top-[0.6rem] tw-right-[0.6rem]',
-                        validation === Validation.Error && 'tw-text-text-negative',
-                        validation === Validation.Warning && 'tw-text-text-warning',
-                    ])}
-                    data-test-id="error-state-exclamation-mark-icon"
-                >
-                    <IconExclamationMarkTriangle />
-                </span>
-            )}
-        </div>
-    );
+export const Textarea = ({ ...props }: TextareaProps): ReactElement => {
+    return <TextareaField {...props} />;
```

Comment:
The new `Textarea` input has new props like `label`, `helpText`, `errorText`. So we are going to split it into `TextareaField` (exported and with stories, this is the current `Textarea` component) and `TextareaInput` (internal, for avoiding breaking changes). The new `Textarea` will verify if the `label` property is provided (required for rendering the `TextareaInput`) and decide which of them should be rendered.